include/openrave/kinbody.h

```diff
@@ -1476,6 +1476,7 @@ class OPENRAVE_API KinBody : public InterfaceBase
         Transform _tRightNoOffset, _tLeftNoOffset;         ///< same as _tLeft and _tRight except it doesn't not include the offset
         Transform _tinvRight, _tinvLeft;         ///< the inverse transformations of tRight and tLeft
         bool _bInitialized;
+        bool _bStatic = false; ///< IsStatic with both lower and upper limits equal 0
```

Comment:
@tgn3000 unfortunately, we don't use initializers like this in openrave headers (even though it is convenient). can you remove? thanks

Reply:
thanks, will address these comments tomorrow